portal/static/js/src/data/common/consts.js

```diff
@@ -0,0 +1,2 @@
+export var EPROMS_SUBSTUDY_ID = 1;
+export var EPROMS_SUBSTUDY_TITLE = i18next.t("IRONMAN sub study");
```

Comment:
>sub study

Is this the preferred format? I've seen `sub-study` and `substudy`, just wanted to double check...

Reply:
thank you, Ivan. I will confirm with Cam.

Reply:
noting here that placeholder text will change per feedback.  A new story will be created to address this.